posts/2019-01-17-Rust-1.32.0.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: post
+title: "Announcing Rust 1.32.0"
+author: The Rust Release Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.32.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.32.0 is as easy as:
+
+```
+$ rustup update stable
```

Comment:
```suggestion
$ rustup update
```

Because this will do a Rustup self-update and specifying a toolchain does not (this is fixed on master, but of course you have to self-update to get the fix :-( )

Reply:
hmmm, so doing this will fail to install stable for anyone who doesn't have stable installed. we've always done it this way in every release post, what's changed since then?

Reply:
@nrc Is a Rustup self-update required for the 1.32 toolchain update?

Reply:
What @nrc means is that `rustup update` does a self update at the end and `rustup update stable` does not. A recent change has happened to do a self-update on any toolchain update but it's not a published version yet.

This is particularly problematic with the 1.31.0 release where a self-update was needed first to get full usage of things like `rustup component add preview`. This in theory is fixed already for anyone coming from the 1.31.0 toolchain, but it will still be broken for anyone coming from a previous 1.30.0 or previous toolchain.

I'd recommend switching this to `rustup update`, they basically mean the same thing anyway

Reply:
I've added a small section suggesting users also update rustup